__tests__/app.test.js

```diff
@@ -28,27 +29,31 @@ describe("GET /api/categories", () => {
             categories.forEach((category) => {
                 expect(category).toHaveProperty('slug', expect.any(String));
                 expect(category).toHaveProperty('description', expect.any(String));
-        })
-    })
-})
-})
-
-//happy request
-//bad requests
-//400, 404
+            });
+        });
+    });
+});
 
-
-// describe("/api/categories", () =>{
-//     test('200: GET responds with category objects with slug and description properties', () => {
-//         return request(app)
-//         .get("/api/categories")
-//         .expect(200)
-//         .then(({ body }) => {
-//             const { categories } = body;
-//             console.log(categories);
-//             categories.forEeach((category) => {
-//                 expect(response.body.categories)
-//             })
-//         })
-//     })
-// });
+describe("GET /api/reviews", () => {
+    test("200: responds with array of review objects", () => {
+        return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then(({ body }) => {
+            const { reviews } = body;
+            expect(reviews.length).toBe(13);
+            reviews.forEach((review) => {
+                console.log(review.comment_count);
+                expect(review).toHaveProperty('owner', expect.any(String));
+                expect(review).toHaveProperty('title', expect.any(String));
+                expect(review).toHaveProperty('review_id', expect.any(Number));
+                expect(review).toHaveProperty('category', expect.any(String));
+                expect(review).toHaveProperty('review_img_url', expect.any(String));
+                expect(review.created_at instanceof Date);
+                expect(review).toHaveProperty('votes', expect.any(Number));
+                expect(review).toHaveProperty('designer', expect.any(String));
+                expect(review).toHaveProperty('comment_count', expect.any(String));
+            });
+        });
+    });
+});
```

Comment:
Great stuff so far, just needs one more test making sure that the responses are ordered by date - take a look at Jest-sorted. It's a super helpful npm package that will let you easily check that the responses are sorted in descending order ( biggest first means the earliest date will come first )